frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -254,7 +254,7 @@ const PlatformWrapper = ({
       <>
         {packageType !== "plain-osquery" && (
           <span className={`${baseClass}__cta`}>
-            Run this command with the{" "}
+            Run this command to generate an install package with the updated{" "}
```

Comment:
> "to generate an install package"

+1 on explaining what running the command will do

Reply:
I think we should stick with the label/help text copy from the [Figma](https://www.figma.com/design/KnUaTpRsHXAu8Cxd9M1K5s/%2322223-Easy-to-understand-fleetctl-package-command-and-output?node-id=0-1&t=V06Gutt5eYeG2shl-1): we settled on keeping the label short but updating the help text to include "Install this package…" to make it clearer what the command does.

Reply:
@rachaelshaw ,
According to the figma file linked here, the text should read : 

1. Install this package to add hosts to Fleet. For CentOS, Red Hat, and Fedora Linux, use --type=rpm. (For windows)
2. Install this package to add hosts to Fleet. For Windows, this generates an MSI package. (For Linux)
3. Install this package to add hosts to Fleet . (For Macos)

Right?